scalabel/label/typing.py

```diff
@@ -45,6 +73,7 @@ class Label(BaseModel):
     box2d: Optional[Box2D]
     box3d: Optional[Box3D]
     poly2d: Optional[List[Poly2D]]
+    graph: Optional[Graph]
```

Comment:
this should rather be of type Optional[List[Graph]]

Reply:
Interesting point. A graph can store two polygons. It should be easy to detect two polygons or poly lines in a graph by id matching. If two adjacent edges are not connected in a graph of a polygon label, there are multiple polygons.

Reply:
So we may not need a list of graph here. Graph is a general container, how to interpret the graph depends on the label type.